docker/airflow/tests/functional_tests/running_test.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 Google LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Here you're using GPL header, Airflow is Apache 2.0 licensed

Reply:
fixed